src/composer/src/build.rs

```diff
@@ -248,12 +250,58 @@ fn constructor_serialize_args(
     Ok(args_file_path)
 }
 
+fn comp_gen_make_cmd_scan_lib_constant(constants: &[String]) -> Result<Vec<String>, String> {
+    println!("Scanning libraries for constants: {:?}", constants); // Print the constants being scanned for
+    let constants_str = constants.join(" ");
+    let output = Command::new("make")
```

Comment:
We're using the `exec_pipeline` in the composer, so I don't see a reason to use two means of constructing pipelines. Either we get rid of the `exec_pipeline`, and use `Command` directly, or we use `exec_pipeline` here.

I might be missing something, and if so, I apologize!

Reply:
we can use exec_pipeline for sure. But I have faced some issue with using exec_pipeline( the command generated from exec_pipeline has some format issue and can not be recognized by make). I need to find a solution for that.